.github/workflows/benchmark.yml

```diff
@@ -0,0 +1,36 @@
+name: Benchmark
+
+on:
+  push:
+    branchs: [ master ]
```

Comment:
`branches`

---

I wonder if it's possible to:

```
on:
  push:
    branches: [ master ]
    paths:
      - 'libs/**'
```

both `branches` and `paths` must match (equivalent to AND).